FindPOLine/utils.js

```diff
@@ -94,7 +101,7 @@ export const getNormalizedISBN = async (isbnNumber, ky) => {
   }
 };
 
-export function getLinesQuery(queryParams, ky) {
+export function getLinesQuery(queryParams, ky, localeDateFormat, customFields) {
```

Comment:
Will `ui-orders` support these changes as well? https://github.com/folio-org/ui-orders/blob/1e101247f8316f8c5eeb02911b59e0ec6683279e/src/OrderLinesList/hooks/useOrderLines/useOrderLines.js#L25

Reply:
Yes. It will work.
However, `getLinesQuery` in `ui-orders` is going to be invoked with the additional parameters as part of [UIOR-1237](https://folio-org.atlassian.net/browse/UIOR-1237).